graphql_compiler/schema_generation/orientdb/schema_graph_builder.py

```diff
@@ -7,18 +7,19 @@
 
 from ..exceptions import IllegalSchemaStateError
 from ..schema_graph import (
-    EdgeType, InheritanceStructure, NonGraphElement, PropertyDescriptor, SchemaGraph, VertexType,
-    get_subclass_sets_from_superclass_sets, link_schema_elements
+    EdgeType, IndexDefinition, InheritanceStructure, NonGraphElement, PropertyDescriptor,
+    SchemaGraph, VertexType, get_subclass_sets_from_superclass_sets, link_schema_elements
 )
 from .schema_properties import (
     COLLECTION_PROPERTY_TYPES, EDGE_DESTINATION_PROPERTY_NAME, EDGE_END_NAMES,
-    EDGE_SOURCE_PROPERTY_NAME, ORIENTDB_BASE_EDGE_CLASS_NAME, ORIENTDB_BASE_VERTEX_CLASS_NAME,
-    PROPERTY_TYPE_LINK_ID, parse_default_property_value, try_get_graphql_scalar_type
+    EDGE_SOURCE_PROPERTY_NAME, ORDERED_INDEX_TYPES, ORIENTDB_BASE_EDGE_CLASS_NAME,
+    ORIENTDB_BASE_VERTEX_CLASS_NAME, PROPERTY_TYPE_LINK_ID, UNIQUE_INDEX_TYPES,
+    parse_default_property_value, try_get_graphql_scalar_type
 )
 from .utils import toposort_classes
 
 
-def get_orientdb_schema_graph(schema_data):
+def get_orientdb_schema_graph(schema_data, index_data=None):
```

Comment:
Why make this optional? Seems like we are adding a lot more surface area we have to test, and I don't see what this is buying us.